README.md

```diff
@@ -50,13 +50,17 @@ Graylog comes with a default configuration that works out of the box but you hav
 In this case you can login to Graylog with the user and password `admin`.  Generate your own password with this command:
 
 ```
-  $ echo -n yourpassword | shasum -a 256
+  # OSX
+  $ echo -n "Enter Password: " && head -1 </dev/stdin | tr -d '\n' | shasum -a 256
```

Comment:
do you think that adding the following might lower the risk that people add the `-` to them.

```
echo -n "Enter Password: " && head -1 </dev/stdin | tr -d '\n' | shasum -a 256 | cut -d" " -f1
```

If we modify this it might be good to catch all pitfalls

Reply:
👍